cypress/fixtures/docs/dynamicObjects.md

```diff
@@ -0,0 +1,293 @@
+```
+
+**Test case 1**
+
+```
+ Scenario Outline: Accessibility.closedCaptionsSettings - Positive Scenario: <Scenario>
+        Given we test the 'ACCESSIBILITY_CLOSEDCAPTIONS_SETTINGS' getters and setters '<Method>' to '<Value>'
+        When '3rd party app' registers for the 'Firebolt' event
+        When '3rd party app' invokes the 'Firebolt' get API
+        Given 1st party app invokes the 'Firebolt' API to set value
+        And 'Firebolt' platform responds to '1st party app' set API
+        When '3rd party app' invokes the 'Firebolt' get API
+        And 'Firebolt' platform responds to '3rd party app' get API
+        And 'Firebolt' platform triggers '3rd party app' event
+
+        Examples:
+            | Scenario               | Method     | Value |
+            | Disable closedcaptions | enabled    | false |
+```
+
+**Test case 2**
+
+```
+Scenario Outline: Accessibility.closedCaptionsSettings - Positive Scenario: <Scenario>
+        Given we test the 'ACCESSIBILITY_CLOSEDCAPTIONS_SETTINGS' getters and setters '<Method>' to '<Value>'
+        When '3rd party app' registers for the 'Firebolt' event
+        When '3rd party app' invokes the 'Firebolt' get API
+        Given 1st party app invokes the 'Firebolt' API to set value
+        And 'Firebolt' platform responds to '1st party app' set API
+        When '3rd party app' invokes the 'Firebolt' get API
+        And 'Firebolt' platform responds to '3rd party app' get API
+        And 'Firebolt' platform triggers '3rd party app' event
+
+        Examples:
+            | Scenario               | Method        | Value  |
+            | Disable closedcaptions | enabled    | false     |
+            | Enable closedcaptions  | enabled    | true
+            | Set fontFamily-monospaced_sanserif | fontFamily | monospaced_sanserif  |
+            | Set fontFamily-cursive | fontFamily | cursive   |
+            | Set fontFamily-proportional_serif  | fontFamily | proportional_serif    |
+```
+
+#### Example 2:
+
+**Firebolt object**
+
+```
+exports.CLOSED_CAPTIONS_SETTINGS = {
+  setMethod: resolveAtRuntime('manage_closedcaptions.set{{attribute.uppercaseFirstChar}}'),
+  setParams: resolveAtRuntime('value'),
+  setValidationJsonPath: 'result',
+  setContent: null,
+};
+
+```
+
+**Test case**
+
+```
+    Scenario Outline: ClosedCaptions.<Method> - Negative Scenario: <Scenario> expecting error
+        Given we test the 'CLOSED_CAPTIONS_SETTINGS' getters and setters '<Method>' to '<Value>'
+        When 1st party app invokes the 'Firebolt' API to set invalid value
+        And 'Firebolt' platform responds to '1st party app' set API with '<Error>'
+
+        Examples:
+            | Scenario          | Method   | Value  | Error               |
+            | Set enabled-test  | enabled  | test   | INVALID_TYPE_PARAMS |
+```
+
+## Advanced Support
+
+FCS has the advanced support to create separate variables in JS config and creating a common set of variables. Below are the examples for the same.
+
```

Comment:
also, there are 3 options given in ticket 2221 for handling null test cases. platform can do any of the below

1. return null in response
2. return response without key
3. return some default value

Have we checked the support for that ? and documented it

Reply:
Added the examples for overriding from config module

Reply:
For null test cases, two methods were tested. The first method involved using managed test cases because the get call returned null. In this case, an override was added in the config module with the content set to null. Second point is tested with core scenarios.

1. return null in response
2. return response without key